index.html

```diff
@@ -140,11 +115,15 @@ <h2 data-key="repo-id"><a href="https://www.github.com/heremaps/{repo-id}" targe
 			$.getJSON('https://api.github.com/orgs/heremaps/repos')
 			.done(function (repo_data) {
 				repo_data.forEach(function (data, index) {
+					/* The GitHub API will return more repos than the ones we expose here.
+						 Target only existing repo sections.*/
 					var $target = $('#repo-info-' + data.name);
 					if ($target.length > 0) {
-						$target.find('[data-role="repo-lastUpdate"]').text(moment(data.updated_at).format('ll'));
-						$target.find('[data-role="fork-count"]').text(data.forks_count);
-						$target.find('[data-role="star-count"]').text(data.stargazers_count);
+						$target.find('[data-key="repo-type"]').text(data.language);
```

Comment:
> I'm confused on the part that shall revert to the JS value after the API has returned null.

That's not done by the code, but a manual step after running with everything hard-coded to `null` once. So my idea is:

- Hard-code the language to `null` everywhere to trigger the API.
- Commit that state in Git.
- Look at the page.
- For entries with an empty string, edit the file again to hard-code whatever language we want.
- Amend the previous commit with the hard-coded languages.